int_https_lb_https_redirect/main.tf

```diff
@@ -59,7 +59,6 @@ resource "tls_private_key" "default" {
 }
 
 resource "tls_self_signed_cert" "default" {
-  key_algorithm   = tls_private_key.default.algorithm
```

Comment:
Why remove?

Reply:
Similar to above, we can continue the conversation there.